block_linux.go

```diff
@@ -74,53 +89,83 @@ func DiskVendor(disk string) string {
 	return strings.TrimSpace(string(contents))
 }
 
+func udevInfo(disk string) (map[string]string, error) {
+	// Get device major:minor numbers
+	devNo, err := ioutil.ReadFile(filepath.Join(pathSysBlock(), disk, "dev"))
+	if err != nil {
+		return nil, err
+	}
+
+	// Look up block device in udev runtime database
+	udevId := "b" + strings.TrimSpace(string(devNo))
+	udevBytes, err := ioutil.ReadFile(filepath.Join(pathRunUdevData(), udevId))
+	if err != nil {
+		return nil, err
+	}
+
+	udevInfo := make(map[string]string)
+	for _, udevLine := range strings.Split(string(udevBytes), "\n") {
+		if strings.HasPrefix(udevLine, "E:") {
+			if s := strings.SplitN(udevLine[2:], "=", 2); len(s) == 2 {
+				udevInfo[s[0]] = s[1]
+			}
+		}
+	}
+	return udevInfo, nil
+}
```

Comment:
The above code is already in master:

https://github.com/jaypipes/ghw/blob/dc696826c46f9a64b1e4c3b7304b0c68e8042a8f/block_linux.go#L77-L100

Could you rebase to master to remove this from this PR?

Reply:
GitHub's UI is just stale here. This is being pulled in from as part of commit 55f8af4 which is already merged, so a further merge would only have one copy. Rebased on top of your merge commit anyway though.